tests/unit/test_zigzag.py

```diff
@@ -155,6 +155,7 @@ def test_missing_build_url_xml(self, missing_build_url_xml, simple_json_config,
             zz = ZigZag(missing_build_url_xml, simple_json_config, TOKEN)
             zz.parse()
 
+    @pytest.mark.skip(reason="This may not be valid anymore")
```

Comment:
Is there a tech debt ticket to revisit this later?

Reply:
https://rpc-openstack.atlassian.net/browse/ASC-1558